Makefile

```diff
@@ -24,11 +24,9 @@ config.yml:
 install: .venv/bin/python .venv/bin/pip-licenses .venv/bin/elastic-ingest
 	.venv/bin/pip-licenses --format=plain-vertical --with-license-file --no-license-path > NOTICE.txt
 
-install-agent: install
-	.venv/bin/pip install -r requirements/agent.txt
-
```

Comment:
I vote we leave this, but just have it alias to `.venv/bin/elastic-ingest` for now. That way the agent code doesn't need to be changed again, and we have a place we can hook into in the future for agent-specific bits when we need/want them. Like:

```
install-agent: .venv/bin/elastic-ingest
    # nothing here
```
WDYT?

Reply:
Sure, just added it back in last commit